internal/controller/install/lookout_controller.go

```diff
@@ -237,7 +237,7 @@ func generateLookoutInstallComponents(lookout *installv1alpha1.Lookout, scheme *
 		}
 	}
 
-	job, err := createLookoutMigrationJob(lookout)
+	job, err := createLookoutMigrationJob(lookout, serviceAccountName)
```

Comment:
Lookout CRD Spec already contains ServiceAccount info (https://github.com/armadaproject/armada-operator/blob/main/api/install/v1alpha1/common.go#L103-L105).

Shouldn't that be reused?